packages/stg-evm-v2/hardhat.config.ts

```diff
@@ -411,7 +411,7 @@ const networks: NetworksUserConfig = {
     },
     'zksync-mainnet': {
         eid: EndpointId.ZKSYNC_V2_MAINNET,
-        url: process.env.RPC_URL_ZKSYNC_MAINNET || 'https://zksync.drpc.org',
+        url: process.env.RPC_URL_ZKSYNC_MAINNET || 'https://mainnet.era.zksync.io',
```

Comment:
Updated since these were failing validation. Also a note that the sandbox rpcs will always fail unless they are up and running

Reply:
Should we then add the `--stage` CLI argument to that task?

Reply:
Ah yes that is a great point -- I will create a follow up ticket to fix the task and try to get to it asap!